am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/GetResourceDefinitionsWithCreatePermissionIntegrationTest.java

```diff
@@ -80,26 +87,26 @@ void shouldNotRetrieveResourceDefinitionWhenRecordExistsWithoutRootAttribute() {
         Set<ResourceDefinition> result =
             service.getResourceDefinitionsWithRootCreatePermission(ImmutableSet.of(ROLE_NAME));
 
-        assertThat(result).isEmpty();
+        assertThat(result).isNull();
```

Comment:
So this comes from AC: 

Given default roles setup doesn't match my roles
When I ask for list of resource definitions I can create based on my roles and security classification
Then I get null

https://tools.hmcts.net/jira/browse/AM-253

I return null when there are no matching records for the roles.
I thought it would also make sense when there are no records with low enough classification.

Reply:
Have changed back to empty list with different implementation - let's discuss tomorrow anyway